_content/monographs/practical-advice-for-meditators_mills-laurence.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Practical Advice for Meditators"
+authors:
+  - mills-laurence
+external_url: "https://www.bps.lk/olib/wh/wh116_Khantipalo_Practical-Advice-For-Meditators.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1kEPSwBL8zktT1XkqHqQ9DJBE35UsQJU7/view?usp=sharing"
+  - "https://drive.google.com/file/d/1kurZQ5objbsMFl2auhKSlXZN2X0FY59I/view?usp=drive_link"
+formats: [pdf, epub]
+course: problems
+tags:
+  - parami
+  - brahmavihara
+  - hindrances
```

Comment:
Yeah: hindrances and daily-life are the subtopics under `problems` so these tags are redundant.